typescript/src/deployers/dispatcher.ts

```diff
@@ -0,0 +1,111 @@
+import type { Deployer, Chain } from "./interface";
+import fs from "fs";
+import path from "path";
+
+import { deployContract, buildAccount } from "../cairo";
+
+const ASSET_CLASS_CRYPTO_ID = 0;
+
+// TODO: This should probably be its own configuration *somewhere*. TBD
+const PRAGMA_FEEDS = [
+  {
+    id: "18669995996566340",
+    name: "BTC/USD: SpotMedian",
+  },
+  {
+    id: "19514442401534788",
+    name: "ETH/USD: SpotMedian",
+  },
+];
+
+// TODO: This should probably be its own configuration *somewhere*. TBD
+const FEED_TYPES = [
```

Comment:
same let's move that to yaml config files

Reply:
done